tests/shared.py

```diff
@@ -136,6 +132,19 @@ def setUp(self):
                             "received {}: {}".format(type(response),
                                                      response))
 
+    def driver_missing_features(self, *features):
+        needed = set(map(lambda f: f.value, features))
+        supported = self._driver_features
+        return needed - supported
+
+    def driver_supports_features(self, *features):
```

Comment:
Sorry, is this actually used somewhere?

Reply:
It will be used in my next round of test migration. Even though, each test should only test one thing, I decided that because spinning up a stub server is relatively expensive, I would add additional assertions that only make sense for driver supporting certain features.

Reply:
Sure, as long as you are confident that you will use it in the next PR and it will not remain unused.